testplan/testing/multitest/base.py

```diff
@@ -80,23 +79,32 @@ def __init__(self):
 
 class RuntimeEnvironment(object):
     """
-    A collection of drivers accessible through either items or named attributes,
-    representing the environment of a Multitest environment instance with runtime
-    information about the currently executing testcase
-
-    This class is a tiny wrapper around the multitest's :class:`Environment`,
-    delegate all calls to it but multitest_runtime_info which serves the runtime information
-    of the current thread of execution.
+    A collection of resources accessible through either items or named
+    attributes, representing a test environment instance with runtime
+    information about the currently executing testcase.
+
+    This class is a tiny wrapper around the :py:class:`Environment` of
+    :py:class:`~testplan.testing.base.Test`, delegates all calls to it
+    but with a `runtime_info` which serves the runtime information of
+    the current thread of execution.
     """
 
     def __init__(
-        self, environment: Environment, runtime_info: MultiTestRuntimeInfo
+        self,
+        environment: entity.Environment,
+        runtime_info: MultiTestRuntimeInfo,
     ):
         self.__dict__["_environment"] = environment
-        self.__dict__["multitest_runtime_info"] = runtime_info
+        self.__dict__["runtime_info"] = runtime_info
 
     def __getattr__(self, attr):
-        return getattr(self._environment, attr)
+        # Instance of `RuntimeEnvironment` is presented in user's testcase
```

Comment:
It is not the purpose of this wrapper to control access to the _environment. I think this could simply be restored to "getattr"

Reply:
simplified